### DISCUSSION TITLE
Custom profiles : Disable preview and Disable intermediate results

### DISCUSSION THREAD

**Discussion by DisasterofPuppets** [lllyasviel/Fooocus#3679] [Q&A] [answered]
Hi all,

I couldn't find the command anywhere to set the below:
Disable Preview
Disable Intermediate results

Well, that's a lie, I see if you use     
"default_black_out_nsfw": false 
it will also check the Disable Preview checkbox, but I don't want the black out NSFW checked.

I tried
default_disable_preview
default_disable_intermediate_results
and
disable_preview
disable_intermediate_results

No luck. I had a look through the html source to see if it was obvious or there were some clues for me there, but no luck.
Is there a way to find the correct references for future fields I may want to turn off that aren't listed in the custom tutorial text file?

Thank you

**Comment by mashb1t** [comment, marked as answer]
You can find all possible config values in the config example file

**Reply by DisasterofPuppets** [reply]
Thank you, Sadly I couldn't find the two for these in the example file. I might be blind?

**Reply by mashb1t** [reply]
you can find it in the same directory as the config.txt and it's automatically generated when you start Fooocus.

**Reply by DisasterofPuppets** [reply]
Cheers, Thank you for such a quick reply. I should have worded my reply better. I found the example file but it doesn't seem to list anything for 

Disable Preview
Disable Intermediate results

I can use 

"default_black_out_nsfw": false which will also check the Disable Preview checkbox, but I don't see a standalone for Disable Preview.

Thanks again

**Comment by TokenChingy** [comment]
This is because it has a restricted value based on performance profile.
